playbooks/byo/preflight/README.md

```diff
@@ -0,0 +1,58 @@
+# Preflight checks
+
+Here we provide an ansible playbook (currently just one) for detecting
+potential roadblocks prior to install or upgrade.
+
+Note that currently this is only useful for RPM-based Enterprise
+installations, based on supported Red Hat RPMs. Containerized and
+Origin installs are excluded from checks for now.
+
+## Running directly
+
+For a simple trial, with an installation of Ansible 2.0 or greater,
+run the playbook directly against your inventory file:
+
+    $ git clone https://github.com/rhcarvalho/openshift-ansible.git -b pre-flight-checks
```

Comment:
We definitely need docs that actually work today so it's too early to refer to the upstream repo.  I'll make a note on Monday to reach out to Productization and see if we could start working from their repo next week.